omod/src/main/web/dashboardwidgets/relationships/relationships.controller.js

```diff
@@ -52,9 +52,11 @@ export default class RelationshipsController  {
 
         //fetchPrivileges
         this.openmrsRest.get('session', {
-            v: 'custom:(privileges)'
+            v: 'custom:(privileges:(name))'
```

Comment:
We've confirmed that it's okay to narrow down this REST call to only return the "name" of the privileges?  (I don't know why that wouldn't be fine, but just double-checking).